scripts/qa_crowbarsetup.sh

```diff
@@ -3810,6 +3819,33 @@ function oncontroller_testsetup()
     nova stop "$instanceid"
     wait_for 100 1 "test \"x\$(nova show \"$instanceid\" | perl -ne 'm/ status [ |]*([a-zA-Z]+)/ && print \$1')\" == xSHUTOFF" "testvm to stop"
 
+    # run tests for Magnum bay deployment
```

Comment:
Good idea, but users will use command line or magnum-UI plugin and I have found already some bugs in it. I would like to have this tests closest to user experience.